tfx/components/example_gen/import_example_gen/executor.py

```diff
@@ -41,18 +41,18 @@ def _ImportSerializedRecord(  # pylint: disable=invalid-name
   Note that each input split will be transformed by this function separately.
 
   Args:
-    pipeline: beam pipeline.
+    pipeline: Beam pipeline.
     exec_properties: A dict of execution properties.
-      - input_base: input dir that contains tf example data.
+      - input_base: input dir that contains TF record data.
```

Comment:
note, TFRecord is a keyword for a file format, 
consider to simply use `contains records` or `contains input data` etc. 
please revisit the usage of this term  (`TF record`, `TFRecord`) in the rest of the change.